packages/ui/src/models/visualization/flows/abstract-camel-visual-entity.ts

```diff
@@ -163,6 +163,48 @@ export abstract class AbstractCamelVisualEntity<T extends object> implements Bas
     }
   }
 
+  isDefiniteNode(path?: string) {
+    if (path === 'route.from' || path === 'template.from') {
+      return false;
+    } else {
+      return true;
+    }
+  }
```

Comment:
```suggestion
  isDefiniteNode(path?: string) {
    return path === 'route.from' || path === 'template.from';
  }
```

Reply:
I'm not exactly sure what we should return here because judging by the method name, it looks like isDefiniteNode would return false for both `from` :thinking: